krakenapi.go

```diff
@@ -91,24 +91,31 @@ const (
 	MinimumUSDT = 5.0
 )
 
-// KrakenApi represents a Kraken API Client connection
-type KrakenApi struct {
+// KrakenAPI represents a Kraken API Client connection
+type KrakenAPI struct {
 	key    string
 	secret string
 	client *http.Client
 }
 
 // New creates a new Kraken API client
-func New(key, secret string) *KrakenApi {
-	return NewWithClient(key, secret, http.DefaultClient)
+func New(key, secret string) *KrakenAPI {
+	krakenAPI := KrakenAPI{
+		key:    key,
+		secret: secret,
+		client: http.DefaultClient,
+	}
+	return &krakenAPI
 }
 
-func NewWithClient(key, secret string, httpClient *http.Client) *KrakenApi {
```

Comment:
Same reasoning here. This looks like a breaking change to me.

Reply:
fixed

Reply:
fixed